<launch>

  <node pkg="data_collector" type="weather_pub.py" name="weather_pub" output="screen"/>
  <node pkg="data_collector" type="Power_status.py" name="Power_status" output="screen"/>
  <node pkg="data_collector" type="space_status.py" name="LS_status" output="screen"/>


</launch>